mono/mini/Makefile.am.in

```diff
@@ -725,9 +728,9 @@ FULLAOT_LIBS = \
 fullaotcheck: $(mono) $(fullaot_regtests)
 	rm -rf fullaot-tmp
 	mkdir fullaot-tmp
-	$(MAKE) fullaot-libs AOT_FLAGS=full GSHAREDVT=$(GSHAREDVT)
+	$(MAKE) fullaot-libs AOT_FLAGS="full$(aot_suffix)" GSHAREDVT=$(GSHAREDVT)
```

Comment:
At the moment, running `mono --aot=full,` with nothing after the comma will always fail with `AOT : Unknown argument ''.`. Maybe that is worth changing however.